bindata/manifests/plugins/sriov-cni.yaml

```diff
@@ -24,6 +24,7 @@ spec:
     spec:
       nodeSelector:
         beta.kubernetes.io/os: linux
+        node-role.kubernetes.io/worker:
```

Comment:
Why this is added? is it related to the operator-sdk update?

Reply:
No. Just want to keep the default nodeSelector aligned with the configure daemon.